### PR TITLE
Adding bn.yml Internationalization

### PR DESCRIPTION
**Pull request by MjAbuz**
DCO Signed off by "@MjAbuz azmikbal@gmail.com
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

kind user-interface

kind content

kind translation

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

area blog

> /

area documentation

Internationalization
> /area videos

**What this PR does / why we need it**: For Internationalization

**Which issue(s) this PR fixes**: Will give more coverage

Fixes #

**Special notes for your reviewer**: DCO signoff

### CODE REVIEW

**Comment by MjAbuz** [comment]
> Welcome @MjAbuz! It looks like this is your first PR to falcosecurity/falco-website 🎉

Yes,

**Comment by MjAbuz** [comment]
Done i guess :)

**Comment by danpopSD** [comment]
/hold

lack of DCO.

File: OWNERS [hunk 1]
```diff
@@ -18,3 +18,4 @@ reviewers:
   - radhikapc
   - leogr
   - Rajakavitha1
+  - MjAbuz
```
[the remark this answers is not in the document]
Comment (MjAbuz) [reply OWNERS:21]: Soory @rikatz i was away and didn't get time to check back that's why didn't reply or respond

File: README.md [hunk 2]
```diff
@@ -74,3 +74,21 @@ To translate other parts of the site:
 
 1. See the `languages.ko` block in the site's [`config.toml`](./config.toml) configuration file and translate those items from the current English.
 1. See [`i18n/ko.yaml`](./i18n/ko.yaml) for a handful of language snippets that need to be translated from the current English. Translate the `other` block of each term.
+
+To translate docs into Japanese, create Japanese-language Markdown files in the `content/ja/docs` directory.
+
+To translate other parts of the site:
+
+1. See the `languages.ja` block in the site's [`config.toml`](./config.toml) configuration file and translate those items from the current English.
+1. See [`i18n/ja.yaml`](./i18n/ja.yaml) for a handful of language snippets that need to be translated from the current English. Translate the `other` block of each term.
```
Comment (leogr) [inline README.md:83]: This seems to be duplicated, and not required here.

**Pull request closed without merging**